pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -240,109 +241,220 @@ func substitueYamlVariables(baseYaml []byte, yamlSubstitutions map[string]string
 	return baseYaml
 }
 
+func getNamedCondition(route *unstructured.Unstructured, conditionTypeValue string) map[string]interface{} {
+	status := route.UnstructuredContent()["status"].(map[string]interface{})
+	conditions := status["conditions"].([]interface{})
+	for i := range conditions {
+		c := conditions[i].(map[string]interface{})
+		klog.V(4).Infof("Condition returned\n%s\n", c)
```

Comment:
should you put this log inside the if?

Reply:
I wanted to be able to see what all the conditions we had were, so had it outside.  Will probably remove later.